Assets/Src/Scripts/Animator/HandLandmarksController.cs

```diff
@@ -87,6 +139,49 @@ public class HandLandmarksController : MonoBehaviour {
       }
     }
 
+    private void ComputeFingerRotation(){
```

Comment:
I think common methods can be extracted from this. 

Essentially this repeats on all 5 fingers, on each finger, update the _fingerTargets's rotation with the LookRotation of the adjacent joints.

You can write a method to process one finger, then call it five times with different start index.

Reply:
Got It， I used the table-driven pattern